queries.js

```diff
@@ -104,16 +104,40 @@ export async function updateStatus(uri, status) {
 }
 
 /**
- * Gets the number of berichtencentrum messages sent since the given time
+ * Gets the number of messages sent by ABB since the given time.
+ * That monitors that the sync with Kalliope and Loket are running.
  */
-export async function getNumberOfSentMessagesSince(time) {
+export async function getIncomingMessagesNumberSince(time) {
   const q = `
     ${PREFIXES}
     SELECT DISTINCT ?message
     WHERE {
       GRAPH ?g {
         ?message a schema:Message ;
-          schema:dateSent ?sentDate .
+          schema:dateSent ?sentDate ;
+          schema:sender <http://data.lblod.info/id/bestuurseenheden/141d9d6b-54af-4d17-b313-8d1c30bc3f5b> .
```

Comment:
This is nitpicking but maybe push this URI for ABB to a constant:
1) easier to re-use 
2) more readable in the code